cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -427,10 +553,46 @@ end
 ###############################################################################
 
 policy "budget_alert" do
+
+  validate $ds_default_dimensions do
```

Comment:
theres no need for these additions, just change the "message" variable in a switch case in lines 528-530. you'll need to update the: 'check eq(xyz)' part of the policy to get an incident to be raised.

Reply:
@lawrethao 
Hey!, I'm just confirming, so you want us to remove both validates (`validate $ds_default_dimensions` and `validate $ds_rbd_dimensions`) and have everything inside the `validate $ds_filter_anomalies`, but to make it work you want us to add the `check`s that were inside those `validate`s in the `validate $ds_filter_anomalies` right?

Reply:
@lawrethao 
Hi Ren, good morning, can you please confirm?

Reply:
These changes were implemented in the last commit: 70292380b084911397e47598172563424c5430cc